packages/core/src/utils/index.ts

```diff
@@ -9,6 +10,10 @@ import type {
 } from "../types.js";
 import { BigNumberishSchema, PercentageSchema } from "./schema.js";
 
+const AlchemyChainMap = new Map<number, Chain>(
+  Object.values(alchemyChains).map((c) => [c.id, c])
+);
```

Comment:
assuming we want to export this:

```suggestion
export const AlchemyChainMap = new Map<number, Chain>(
  Object.values(alchemyChains).map((c) => [c.id, c])
);
```